.github/ISSUE_TEMPLATE/issue_review_template.md

```diff
@@ -10,18 +10,29 @@ assignees: ''
 **Release PR or issue to review**
 <!-- Put here the link(s) to the issue(s) or PR(s) that need to be reviewed -->
 
-**Reviewer actions**
+**Review actions**
 
-To review, assign this issue to yourself and follow the below list.
-For any item, review the file(s) in the issue/PR listed above. 
-Put comments in the above issue or PR if they concern non-changed text.
-When done, tick the box in this issue. 
+Assign this issue to yourself and follow the below list.
+For any review action, review the file(s) in the issue/PR listed above. 
+Put comments in the above issue or PR if they concern non-changed files/text.
 
 - [ ] API definition files (YAML) (check version in info & servers objects)  
 - [ ] test file(s) availability
 - [ ] changelog updated
 - [ ] readme updated (enforce correct release number / API version naming) 
 - [ ] API readiness checklist(s)
 
+**Release actions**
+
+Assign this issue to yourself or another RM team member and follow the below list. 
+When done, tick the box in this issue. 
+
+- [ ] API release tracker available on wiki
+- [ ] Review comments addressed (by Sub Project)
+- [ ] Release PR approved (on behalf of Release Management)
+- [ ] PR merged (by Sub Project codeowner)
+- [ ] Release created within GitHub (by Sub Project codeowner)
+- [ ] Release Tracker updated (wuth creation date of the release)
```

Comment:
```suggestion
- [ ] Release Tracker updated (with creation date of the release)
```

Reply:
Have corrected that separately - so done